gh-132416: document behavior of Path.symlink_to() when link or/and target already exist (#132704)
python/cpython: Doc/library/pathlib.rst

   as a directory if *target_is_directory* is true or a file symlink (the
   default) otherwise.  On non-Windows platforms, *target_is_directory* is ignored.

>    If the link or/and the target already exist, :exc:`FileExistsError` is raised.

Reviewer: The target can exist, so:

```suggestion
   If this path already exists, :exc:`FileExistsError` is raised.
```
